advisor/src/main/kotlin/Advisor.kt

```diff
@@ -65,12 +65,16 @@ class Advisor(
             return ortResult
         }
 
+        val packages = ortResult.getPackages(skipExcluded).map { it.pkg }
+        if (packages.isEmpty()) {
+            log.info { "Skipping advisor run, as there are no packages to advise." }
```

Comment:
Nits:
- I'd drop the common.
- Maybe better: "...no packages to give advice for."